firecrawl_handler.py

```
@@ -60,19 +67,24 @@ async def scrape_url_content(url: str) -> Optional[str]:
         return markdown_content
 
     except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)